day-one/create-resource-from-service.mdx

```diff
@@ -1,6 +1,200 @@
 ---
 title: 'Create Resource from Service'
 sidebarTitle: 'Create from a Service'
-description: 'Guidance for enterprise developers on creating services using Amplication, including .NET and Node.js options.'
+description: "Quickly generate a production-grade service using our opinionated, pre-configured Node.js or .NET code generation engines."
 icon: 'code-simple'
----
\ No newline at end of file
+---
+
+<Frame>
+  <img src="/images/day-one/service.png" />
+</Frame>
+
+You may already have established technology standards, but Amplication provides an additional resource option called a **Service**.
+Services generate a complete, opinionated codebase in Node.js or .NET that you can customize extensively to fit into your existing software ecosystem.
+
+Service resources use Amplication's best-in-class Node.js and .NET code generation engine to produce human-readable, best-practice code.
+If you're comfortable with Node.js or .NET, or need to spin up a backend quickly, creating a service is a great fit.
+
+<Note>
+  **Blueprints vs Services**: Blueprint resources are technology-agnostic,
+  generating code for virtually any language or framework using private plugins.
+  Services, on the other hand, use Amplication's opinionated standards.
```

Comment:
Amplication opinionated services are based on pre-configured templates and technologies in .NET or Node.js. The services can be configured from various plugins for each technology that available out of the box.

Using blueprints you can build your own standards and templates in any language or technology.